packages/app/src/cli/services/dev/extension/bundler.ts

```diff
@@ -27,6 +27,9 @@ export async function setupBundlerAndFileWatcher(options: FileWatcherOptions) {
 
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
   options.devOptions.extensions.forEach(async (extension) => {
+    if (extension.type === 'flow_template') {
+      return
+    }
     bundlers.push(
```

Comment:
I'm early returning here because otherwise the extension build fails. I believe this is because the "previewable" flag `ui_preview` is mainly catered towards ui extensions and Flow templates don't have the same requirements.

Reply:
Yes, but instead of explicitly checking the type here, you can filter the extensions with `isESBuildExtension`:

Something like:
```ts
const bundableExtensions = options.devOptions.extensions.filter((ext) => ext.isESBuildExtension)
```.

Until now, all Previewable extensions where also ESBuildExtensions, that's no longer true so we need to add this extra filter.